kotlin/example/demo-android-app/app/src/main/java/com/example/myapplication/MainActivity.kt

```
@@ -1,185 +1,281 @@
+            val receivedAt = System.currentTimeMillis()
+            val transferTime = receivedAt - sentAt
+
+            Log.d(TAG, "Verifier received TEXT=$actualMsg")
+            Log.d(TAG, "Transfer time = $transferTime ms")
+
+            runOnUiThread {
+              findViewById<TextView>(R.id.transferText)?.text =
+                "Transfer Time: $transferTime ms\n\nMessage: $actualMsg\n\nTransfer Time: ${transferTime} ms"
+            }
+
+          } catch (ex: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)